services/mailer.js

```diff
@@ -54,102 +54,108 @@ templates.render = (name, format = 'txt', context) => new Promise((resolve, reje
 
     return resolve(view(context));
   });
-}); // ends templates.render
+});
+
+const mailer = {};
+
+// enabled is true when the required configuration is available. When testing
+// is enabled, we will be simulating that emails are being sent, because in a
+// production system, emails should and would be sent.
+mailer.enabled = Boolean(
+  SMTP_HOST && SMTP_HOST.length > 0 &&
```

Comment:
Isn't it enough to just check `SMTP_HOST` instead of `SMTP_HOST && SMTP_HOST.length > 0`?